python/tests/sql_aggregates/test_every.py

```diff
@@ -0,0 +1,70 @@
+import unittest
+from feldera import PipelineBuilder, Pipeline
+from tests import TEST_CLIENT
+
+class TestAggregatesBase(unittest.TestCase):
+    def setUp(self) -> None:
+        self.data = [{"insert":{"id": 0, "c1": 5, "c2": 2, "c3": None, "c4": 4, "c5": 5, "c6": 6, "c7": None, "c8": 8}},
```

Comment:
most of the columns here are not used, you can perhaps remove them?
unless the table schema is required by the test structure

could have a boolean column for this test